src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs

```diff
@@ -62,16 +62,6 @@ private void OnLoaded(object sender, RoutedEventArgs e)
             FocusManager.SetFocusedElement(LibrarySearchBox, LibrarySearchBox);
         }
 
-        private void IncludeAllLibraryFilesRb_Checked(object sender, RoutedEventArgs e)
```

Comment:
What code is responsible now for making sure the checkboxes remain checked when "IncludeAllLibraryFiles" is selected?

Reply:
I was going to flag this too - seems like it should be a binding, not an event handler.

Reply:
IsChecked is bound to treeViewItem checkbox. And the value is set here base on selection type:
https://github.com/aspnet/LibraryManager/blob/285984f683c213c7f50dfcdb6542fac761e82cf3/src/LibraryManager.Vsix/UI/Models/PackageItem.cs#L41